src/runtime/vm/vm.cc

```diff
@@ -641,11 +714,21 @@ void VirtualMachine::Run() {
         pc++;
         goto main_loop;
       }
+      case Opcode::Fatal: {
+        throw std::runtime_error("VM encountered fatal error");
+      }
       case Opcode::LoadConst: {
         WriteRegister(instr.dst, this->constants[instr.const_index]);
         pc++;
         goto main_loop;
       }
+      case Opcode::LoadConsti: {
+        auto tensor = NDArray::Empty({1}, {kDLFloat, 32, 1}, {kDLCPU, 0});
```

Comment:
Should it be `kDLInt`?

Reply:
good catch! fixed.

Reply:
I think there are a few other places after this one. Please check and fix as well.